lib/pages/invite-error-page.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+
+import By from 'selenium-webdriver';
```

Comment:
It looks like this import should be changed to:
`import { By } from 'selenium-webdriver';`

Reply:
indeed it was this, thanks for the review!